cmd/ical-notifier/misc/ical-notifier.cron

```diff
@@ -0,0 +1 @@
+0,15,30,45 * * * * ical-notifier --config /etc/ical-relay/noitifer-config.yml > /proc/1/fd/1 2>/proc/1/fd/2
```

Comment:
`*/15 * * * *` might be a bit nicer solution here

Reply:
Anyways, isn't this superseeded by the systemd timer?

Reply:
```suggestion
*/15 * * * * ical-notifier --config /etc/ical-relay/noitifer-config.yml > /proc/1/fd/1 2>/proc/1/fd/2
```

ja das ist natürlich schöner.

Reply:
> Anyways, isn't this superseeded by the systemd timer?

für unser deb package zum installieren ja. aber docker hat ja kein systemd-timer, deswegen nutzen wir in docker cron